### PR TITLE
Create Launch Screen Views

### LINKED ISSUE #8: Design Views for Launch, Log In, Registration

### PR DESCRIPTION
**Pull request by Vielheim**
Set up views for Launch, Log In and Registration

Set up custom TextField and SecureField (for consistent styling between the launch views)

Merged from main branch, resolved the merge conflicts in project.pbxproj

### CODE REVIEW

**Comment by Vielheim** [comment]
I think I created two issues on the Project board by accident. Linked them together, will just resolve when we merge

File: TourMate/TourMate/Launch/RegisterView.swift [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+                    InputSecureField(title: "Password *", textField: $password)
+
+                    InputSecureField(title: "Confirm Password *", textField: $confirmPassword)
+
+                    if showWarning {
+                        Text(warningMessage)
+                            .foregroundColor(.red)
+                            .padding()
+                    }
+
```
Comment (keanecjy) [inline TourMate/TourMate/Launch/RegisterView.swift]: I think the warning should be below, otherwise it shifts the above fields up

**Review by keanecjy — APPROVED** [review]
Looks good, small nit to fix

File: TourMate/TourMate/Launch/RegisterView.swift (hunk 1 above)
Comment (Vielheim) [reply TourMate/TourMate/Launch/RegisterView.swift]: Below the Button?

Comment (keanecjy) [reply TourMate/TourMate/Launch/RegisterView.swift]: yes

Comment (Vielheim) [reply TourMate/TourMate/Launch/RegisterView.swift]: Ok! I shifted it

**Pull request closed without merging**